### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by sagar-barapatre** [Namanl2001/MERN-Gurujii-dev#4]
It's not required now but in future we can implement the feature of dark mode on our website. So that user to toggle the button to switch to different themes.

**Comment by Namanl2001** [comment]
Thanks! again @sagar-barapatre for opening this issue. 
In general, I see a dark theme as a nice-to-have, but we have to weigh the costs. My main reservations are:

- It will be more work to implement than it may seem at first. CSS styles are in inline styles in the JSX.
- We also need to make sure that everything still looks ok in both light and dark modes after any new addition or change.
- visual testing is more difficult.

None of that is a major problem, if someone wants to contribute to this, I'm happy to help.

**Comment by sagar-barapatre** [comment]
Yeah! I think it's a big task, so let this issue be open. This issue will maybe require a team of 3-4 people who can review over the feature.

**Comment by thaditi** [comment]
Hey @Namanl2001 and @sagar-barapatre. I would like to work on this issue as part of GSSOC'21.

**Comment by khareyash05** [comment]
Hey @Namanl2001  and @sagar-barapatre .I would like to work on this issue.If no one else is assigned.I will work on this issue as a part of GSSOC'21

**Comment by sagar-barapatre** [comment]
Actually this issue might require 2-3 people for testing this feature. So shall we together work on this issue ?
What say @Namanl2001 @thaditi @khareyash05 ??

**Comment by khareyash05** [comment]
@sagar-barapatre I would love to

**Comment by thaditi** [comment]
@sagar-barapatre Sure.

**Comment by Namanl2001** [comment]
Great! now as we have a team of three I think you all can start discussing your ideas and approach with each other. Here on the issue or in the discussions https://github.com/Namanl2001/MERN-Gurujii-dev/discussions.

Hopefully we would have a separate discord channel for our project and a project mentor with us soon, provided by the GSSoC'21 team.
Thank you @thaditi @khareyash05 @sagar-barapatre for taking up this issue.

All the best!!

**Comment by sagar-barapatre** [comment]
> Great! now as we have a team of three I think you all can start discussing your ideas and approach with each other. Here on the issue or in the discussions https://github.com/Namanl2001/MERN-Gurujii-dev/discussions.
> 
> Hopefully we would have a separate discord channel for our project and a project mentor with us soon, provided by the GSSoC'21 team.
> Thank you @thaditi @khareyash05 @sagar-barapatre for taking up this issue.
> 
> All the best!!

P.S. - I m already a mentor

**Comment by khareyash05** [comment]
I have added a **dark mode toggle.** on the bottom right corner.Its a perfect working one and we can also **isolate those elements whose styling is not to be changed**....Also it **remembers the mode(dark/light)** when the last time u opened it.
Suggest me with some changes
![2021-03-07](https://user-images.githubusercontent.com/60147732/110231306-0d6e5800-7f3d-11eb-896d-a33ffeae773f.png)

**Comment by Namanl2001** [comment]
@khareyash05 open a PR then it would be easy to review it.

**Comment by khareyash05** [comment]
I have opened a PR kindly review the same

**Comment by sagar-barapatre** [comment]
Can you place the toggle button just besides the Google signin button, coz generally dark mode buttons are place at top right corners in almost every website.
This is just a suggestion.

**Comment by ritik307** [comment]
@sagar-barapatre @Namanl2001 @khareyash05  is anyone working on this issue. If not then can you plz assign this issue to me?

**Comment by khareyash05** [comment]
@ritik307 I am working on the issue

**Comment by Jassi10000** [comment]
@Namanl2001 Is the issue still open , being a GSSOC'21 participant I want to try my hands out on this issue

**Comment by sagar-barapatre** [comment]
@Jassi10000 PR has already been made. So you can suggest some changes which you would like to if you want.

**Comment by Jassi10000** [comment]
Sure

**Comment by khareyash05** [comment]
@sagar-barapatre 
![2021-03-31 (1)](https://user-images.githubusercontent.com/60147732/113135836-451ca700-9240-11eb-9e07-56b3e61cace9.png)
IS this our website? IF no pls tell what have i done wrong

**Comment by khareyash05** [comment]
![2021-03-31 (6)](https://user-images.githubusercontent.com/60147732/113136244-c4aa7600-9240-11eb-95d2-3d315c913a24.png)
@sagar-barapatre Is this how our website looks?
If no pls tell what have i done wrong?

**Comment by khareyash05** [comment]
![2021-03-31 (7)](https://user-images.githubusercontent.com/60147732/113140022-777cd300-9245-11eb-9ccd-fa118d7619c5.png)
@sagar-barapatre @Namanl2001 I am getting this error Kindly help me

**Comment by sagar-barapatre** [comment]
set hidden : false
The error gets removed

**Comment by subhangi2731** [comment]
@Namanl2001 @sagar-barapatre please assign me this I want to work on this

**Comment by sagar-barapatre** [comment]
Okay assigned.
